otm-bridge.html

```diff
@@ -748,7 +767,7 @@ <h3><dfn>List Deletes</dfn></h3>
         <p>Retrieves a listing of all in-process deletes</p>
         <ul>
           <li>Request Type: <code>GET</code></li>
-          <li>Request Path: <code>/delete ? status=</code></li>
+          <li>Request Path: <code>/delete ? status</code></li>

```

Comment:
```suggestion
          <li>Request Path: <code>/delete ? status</code></li>
```

Reply:
Looks like the restored equals is missing here